app/forms/DateOfBirthFormProvider.scala

```diff
@@ -31,6 +32,8 @@ class DateOfBirthFormProvider @Inject() extends Mappings {
         allRequiredKey = "dateOfBirth.error.required.all",
         twoRequiredKey = "dateOfBirth.error.required.two",
         requiredKey    = "dateOfBirth.error.required"
-      )
+      ).verifying(maxDate(today, "dateOfBirth.error.futureDate", formatDateToString(today)))
+       .verifying(minDate(LocalDate.of(1900,1,1),"dateOfBirth.error.pastDate"))
```

Comment:
why?

Reply:
why are we adding additional validation?

Reply:
where is this requirement coming from?

Reply:
I'd assume it came from the back of initial testing when the page was originally created then went through design?

would we want to allow users to put a future date in for a date of birth? or a date that is too far in the past?

Reply:
It's fine as long as its standard across dates entered and cleared with everyone, the granny in the shed could be over 120 years old.

Reply:
hahaha.. yeah Maddie checked with content team. The last granny born in the 1800's died in 2017. God rest her soul.